packages/term/ansi_term.pony

```diff
@@ -14,6 +22,24 @@ type _EscapeState is
   | _EscapeMod
   )
 
+class _TermResizeNotify is SignalNotify
```

Comment:
formatting is usually blank line between fields and methods and newlines between methods, can you add the 2 newlines here? one after line current line 26 and one after current line 28?